data/mapcss-mapping.csv

```diff
@@ -96,7 +101,8 @@ leisure|pitch;69;
 highway|pedestrian;70;
 natural|scrub;71;
 highway|motorway|bridge;[highway=motorway][bridge?];;name;int_name;72;
-deprecated|deprecated;73;x
+# -service includes all service=siding/yard/crossover except =spur (see replaced_tags.txt).
+railway|rail|service;[railway=rail][service=service];;name;int_name;73;
```

Comment:
UPD: Put a comment here that service=service goes from replace_tags.txt
And it has more than 1M entries in total.

Reply:
but there is a comment already?

Reply:
Ok :)